code/scripts/mdBook_assets.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
```

Comment:
Instead of the file being called `mdbook_assets.sh`, it should be something along the lines of `copy_example_assets.sh` because it can copy other things too.

Reply:
Fixed in https://github.com/JacquesCarette/Drasil/pull/3848/commits/a336d65dc8bfea49f692102e0915753837537b50.